src/Request/uapi-parser.js

```diff
@@ -322,4 +341,4 @@ Parser.prototype.parse = function (xml) {
   });
 };
 
-module.exports = Parser;
+module.exports = { Parser, defaultConfig, errorsConfig };
```

Comment:
This will do:

```javascript
export function defaultConfig() { ... }
export function errorsConfig() { ... }
export default function Parser() { ... }
```